scripts/gen_single_image_diffusion.py

```diff
@@ -435,6 +490,96 @@ def generate(
         help="Canny thresholds",
     )
 
+    options.parser.add_argument(
```

Comment:
Let's not repeat the options here as much as possible, and reuse the one from `palette_model.py` ?

Reply:
I put them here because I think we might want to modify those parameters at inference time (to detect more or less edges)